prog/postgres/postgres_timeline_nexus.rb

```diff
@@ -64,13 +64,19 @@ def before_run
   end
 
   label def wait
+    leader = postgres_timeline.leader
```

Comment:
These should come after `postgres_timeline.blob_storage.nil?` check.

Reply:
if we move these checks after the postgres_timeline.blob_storage.nil? check, then this logic would not clean the dev environments without minio clusters (such as mine). `postgres_timeline.backups` call also has `postgres_timeline.blob_storage.nil?` check inside.